cmd/abapEnvironmentPullGitRepo.go

```diff
@@ -76,8 +76,6 @@ func runAbapEnvironmentPullGitRepo(options *abapEnvironmentPullGitRepoOptions, t
 		return fmt.Errorf("Something failed during the pull of the repositories: %w", err)
 	}
 
-	log.Entry().Info("-------------------------")
-	log.Entry().Info("All repositories were pulled successfully")
```

Comment:
I think this is the right place for finishPullLogs()

Reply:
Thatswhere the duplication came from. These logs are equivalent to `finishPullLogs()`, that's why I removed them.